cmake/packaging.cmake

```diff
@@ -6,7 +6,7 @@
 # the LICENSE file found in the root directory of this source tree.
 #
 
-set(PACKAGE_VERSION 1)
+set(PACKAGE_VERSION 2)
```

Comment:
This is bumped because we change the `INCLUDES DESTINATION` in the install interface